typescript/framework-extensions/langchain/package.json

```diff
@@ -41,11 +41,7 @@
   ],
   "dependencies": {
     "@coinbase/agentkit": "^0.2.2",
-    "@coinbase/coinbase-sdk": "^0.15.0",
```

Comment:
the langchain package doesn't actually use coinbase-sdk, so this can be removed cc @CarsonRoscoe @John-peterson-coinbase

Reply:
Yes. Let's drop it